### ISSUE TITLE
Black Screen on MSI GP60 2PE Leopard Unless Using Graphical Fallback Mode

### ISSUE THREAD

**Issue by FluorescentGreen5** [rescuezilla/rescuezilla#257]
Rescuezilla seems to hang on a black screen on an MSI GP60 2PE Leopard unless I use Graphical Fallback mode. I am using YUMI from the thumbdrive. This hasn't happened with previous versions of Rescuezilla.

**Comment by shasheene** [comment]
Yep, there has been many reports that default "Hirsute" version of Rescuezilla v2.2 has trouble reaching the desktop on some hardware.

As you know, one option is to try choosing the 'Graphical Fallback Mode' item from the boot menu (after selecting a language). **But another alternative is to try the Rescuezilla v2.2 version based on the Ubuntu 20.04 (Focal) operating system which may be able to boot more reliably than the default Ubuntu 21.04 version. You can find the 'Focal' version [at the bottom of the GitHub release page under the 'Assets' section](https://github.com/rescuezilla/rescuezilla/releases/latest).**

The more recent Linux kernels that come with more recent versions of Ubuntu improve support for more recent hardware, but in the case of "Hirsute" it has introduced some compatibility issues. Rescuezilla v2.3 (scheduled for November 2021) should resolve this by (in part) switching the primary release to the upcoming Ubuntu 21.10 "Impish", which will have an upgraded Linux kernel will address some boot issues introduced by Ubuntu 21.04 (Hirsute).

I should also note for anybody interested: Rescuezilla v2.3 will also introduce several ISO images based on the recently released Debian 11 ("Bullseye") version, but this is not expected to improve compatibility beyond the same upgraded Linux kernel that Ubuntu 21.10 has anyway.

tldr:  Try the 'Focal' version [at the bottom of the GitHub release page under the 'Assets' section](https://github.com/rescuezilla/rescuezilla/releases/latest). It should boot more reliably your machine.

**Comment by FluorescentGreen5** [comment]
I tested focal on the same machine, and it worked without graphical fallback mode. Thanks!
